player
diff -r 8f824d68fad6 src/game/Handlers/DuelHandler.cpp  
--- a/src/game/Handlers/DuelHandler.cpp    Fri Dec 04 04:13:28 2009 +0200  
+++ b/src/game/Handlers/DuelHandler.cpp    Sun Dec 06 20:54:56 2009 +0300  
@@ -51,6 +51,15 @@  
       player->duel->startTimer = now;  
       plTarget->duel->startTimer = now;  
     
+    // reset cooldowns and HP/Mana  
+    player->SetHealth(player->GetMaxHealth());  
+    plTarget->SetHealth(pl->GetMaxHealth());  
+  
+    if (player->getPowerType() == POWER_MANA) player->SetPower(POWER_MANA, player->GetMaxPower(POWER_MANA));  
+    if (plTarget->getPowerType() == POWER_MANA) plTarget->SetPower(POWER_MANA, plTarget->GetMaxPower(POWER_MANA));  
+    //only for cooldowns which < 15 min  
+    if (!player->GetMap()->IsDungeon()) { player->RemoveArenaSpellCooldowns(); plTarget->RemoveArenaSpellCooldowns(); }  
+  
       player->SendDuelCountdown(3000);  
       plTarget->SendDuelCountdown(3000);  
   }
